packages/components/lib/utils/platform_utils.dart

```diff
@@ -0,0 +1,23 @@
+import 'dart:io' show Platform;
+import 'package:flutter/foundation.dart' show kIsWeb;
+
+class PlatformUtils {
```

Comment:
Some docs missing + missing the `Source: <stack overflow link>` you should always link other code that it is not yours